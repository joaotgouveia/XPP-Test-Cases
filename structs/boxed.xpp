let a = { b = box(1) };
a.b := 30;
*(a.b);;

let a = { b = box({ c = 1 })};
a.b := { d = 1 };
*(a.b).d;;

let a = box({ b = 1 });
a := { b = 2 };
*a.b;;
