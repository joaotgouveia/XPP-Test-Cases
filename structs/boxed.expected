Value: 30
Type: int

Value: 1
Type: int

Value: 2
Type: int
